.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
```

Comment:
-  [ ] I couldn't find the video. Please be reminded that you should record yourself explaining the main features of your project

Reply:
I have added it, you can find it here --> [Video](https://www.loom.com/share/c592bbb85aec4eb3b01e5d6ad0daf4d9)

Reply:
Well solved